### DISCUSSION TITLE
Deactivate printing for participants

### DISCUSSION THREAD

**Discussion by antoniapsy** [REDCap-German-User-Group/Diskussion-FAQ#6] [Fragen und Antworten]
Hello everyone,

We are planning an online questionnaire survey. A publisher's licence agreement requires that participants cannot print out the questionnaire in order to use it. For this purpose, the option to print should be completely deactivated for the participants or, if it is possible, a white sheet should be printed. Is there a way to implement this in Redcap? I look forward to hearing from you.

Best regards Antonia

**Comment by grezniczek** [comment]
@antoniapsy - When something is shown in a browser, there is no way to prevent it from being printed - you can only make it tedious for the person wanting to print it. But they can alway just screenshot stuff.

To prevent printing, add a CSS rule to the survey, in any label, even of a hidden field. Add:
```css
<style>
@media print {
  body {
    display: none !important;
  }
}
</style>
```
This will produce a blank page.
